### ISSUE TITLE
cmd/govim: investigate gopls.Definition flake

### ISSUE THREAD

**Issue by myitcv** [govim/govim#507]
```
--- FAIL: TestScripts (17.71s)
    --- FAIL: TestScripts/scripts (0.00s)
        --- FAIL: TestScripts/scripts/go_to_def_useopen (0.52s)
            testscript.go:308: 
                # Test that GOVIMGoToDef works
                # More natural to split below and to the right (0.027s)
                # Definition in an open window in current tab (0.252s)
                > vim ex 'set switchbuf=useopen'
                > vim ex 'e '$WORK/q/q.go
                > vim expr '[winnr(), tabpagenr()]'
                [stdout]
                [1,1]
                > stdout '^\Q[1,1]\E$'
                > vim ex 'split '$WORK/p.go
                > vim ex 'call cursor(5,17)'
                > vim ex 'GOVIMGoToDef'
                [stderr]
                unexpected command error: failed to ex(GOVIMGoToDef) in Vim: Caught 'got error whilst handling command:GOVIMGoToDef: failed to call gopls.Definition: no CheckPackageHandles for file://$WORK/q/q.go
                params were: &protocol.TextDocumentPositionParams{
                    TextDocument: protocol.TextDocumentIdentifier{URI:"file://$WORK/p.go"},
                    Position:     protocol.Position{Line:4, Character:16},
                }' in function <SNR>17_define[40]..<SNR>17_callbackCommand, line 5
                [exit status 255]
                FAIL: testdata/go_to_def_useopen.txt:14: unexpected command failure
```

Full build log https://gist.github.com/myitcv/5e2404143be2e2ff530ae8930129f29e

This is a `gopls` error which has been reported as https://github.com/golang/go/issues/34131

**Comment by myitcv** [comment]
Marking as "onhold" because ultimately this is caused by `gopls` bug

**Comment by myitcv** [comment]
Closing this. If the bug is fixed, great, otherwise we need to wait for #591
